src/bikesList.tsx

```diff
@@ -157,25 +168,25 @@ const  BikeListPage = () => {
               </ListSubheader>
               {list.map((bike,index)=>{
                 return (         
-                    <li key={bike.ID}>
+                    <li key={bike.id}>
                     <ListItem style={{backgroundColor: '#69696e',color:'white',display:'flex'}} onClick={()=>handleListItemClick(index)}>
                       <Box display="flex" flexDirection="row" p={1} m={1} alignSelf="center" style={{width:'90%'}}>
                         <Box p={0} width="5%"  >
-                          <ListItemText primary={bike.ID} ></ListItemText>
+                          <ListItemText primary={bike.id} ></ListItemText>
                         </Box>
                         <Box p={0} width="10%">
-                          <ListItemText primary={BikeState[bike.State]}  ></ListItemText>
+                          <ListItemText primary={BikeState[bike.status]}  ></ListItemText>
                         </Box>
                         <Box p={0} width="5%">
-                        <ListItemText primary={bike.StationID} ></ListItemText>
+                          <ListItemText primary={bike.station==null? "unsigned":"bike.station"} ></ListItemText>
```

Comment:
interface Bike powinien raczej wyglądać tak:
```
export  interface Bike{
    id: string;
    status: BikeState;
    station?: { 
        id: string;
        name: string;
    };
  }
```
(zgodnie ze specyfikacją GET /bikes).
Wtedy tutaj możemy dać:
```
{bike.station==null? "" : bike.station.name }
```
(pusty string podoba mi się bardziej niż "unsigned", czy "moving", ale to już opinia).